sw-emulator/app/src/main.rs

```diff
@@ -397,6 +395,8 @@ fn upload_fw_to_mailbox(mailbox: &mut MailboxInternal, firmware_buffer: Rc<Vec<u
     let mut soc = mailbox.as_external();
     // Write the cmd to mailbox.
 
+    assert_eq!(soc.read(RvSize::Word, OFFSET_LOCK).unwrap(), 0);
```

Comment:
Is this for debug ?

Reply:
No need to fix the firmware read complete in mailbox.rs?

Reply:
Can we add Vishal's "make run" to the CI? We are not covering emulator app and we risk breaking it again.

Reply:
Done